contrib/test/integration/test.yml

```diff
@@ -21,5 +21,5 @@
   shell: "CGROUP_MANAGER=cgroupfs STORAGE_OPTIONS='--storage-driver=overlay{{ extra_storage_opts | default('') }}' make localintegration >& {{ artifacts }}/testout.txt"
   args:
     chdir: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-incubator/cri-o"
-  async: 5400
-  poll: 30
+  async: '{{ 2 * 60 * 60 }}'
```

Comment:
I think [one hour or so][1] would be better.

[1]: https://github.com/kubernetes-incubator/cri-o/issues/1424#issuecomment-371027181

Reply:
@wking I think it best to keep timeout changes in a separate PR, was only tickling a bug here temporarily...I'll open a new one....

Reply:
Opened: https://github.com/kubernetes-incubator/cri-o/pull/1429